pkgs/by-name/go/gossip/package.nix

```diff
@@ -64,10 +65,12 @@ rustPlatform.buildRustPackage rec {
     openssl
   ] ++ lib.optionals stdenv.isDarwin [
     darwin.apple_sdk.frameworks.AppKit
+    darwin.apple_sdk.frameworks.Cocoa
     darwin.apple_sdk.frameworks.CoreGraphics
     darwin.apple_sdk.frameworks.Foundation
   ] ++ lib.optionals stdenv.isLinux [
     wayland
+    wayland-scanner
```

Comment:
in `buildInputs`? `wayland-scanner` is used in build process I suppose?

Reply:
Correct, thanks for noticing!